src/hooks/use-site-details.tsx

```diff
@@ -217,17 +221,23 @@ export function SiteDetailsProvider( { children }: SiteDetailsProviderProps ) {
 					}
 					return prevSelectedSiteId;
 				} );
+				// It replace the temporary site created in React
+				// with the new site generated in the backend, but keep isAddingSite to true
```

Comment:
```suggestion
				// with the new site generated in the backend, but keeps isAddingSite to true
```

Just fixing up some grammar 😅